web-console/src/views/services-view/services-view.tsx

```diff
@@ -67,14 +67,25 @@ const allColumns: string[] = [
   'Current size',
   'Max size',
   'Usage',
+  'Start time',
   'Detail',
   ACTION_COLUMN_LABEL,
 ];
 
 const tableColumns: Record<CapabilitiesMode, string[]> = {
   'full': allColumns,
   'no-sql': allColumns,
```

Comment:
ok, so since you are not updating `/druid/coordinator/v1/servers?simple` you should make sure that `Start time` is no in the `no-sql` column list. `no-sql` is the mode used when the user had no SQL access so it falls back to the old endpoint. You should set `no-sql` to:

```
[
  'Service',
  'Type',
  'Tier',
  'Host',
  'Port',
  'Current size',
  'Max size',
  'Usage',
  'Detail',
  ACTION_COLUMN_LABEL,
];
```

Also at this point you can inline the `allColumns` constant as it's reasons for existence was to set `full` and `no-sql` to the same thing.